File: docs/source/_src_snippets/simulation/environment_setup/environment_models/kepler_ephemeris.py
initial_state_in_keplerian_elements = ...

epoch_of_initial_state = ...
central_body_gravitational_parameter = ...

frame_origin = "SSB"
frame_orientation = "J2000"

body_settings.get_body( "Jupiter" ).ephemeris_settings = environment_setup.ephemeris.keplerian( initial_state_in_keplerian_elements,
	epoch_of_initial_state, central_body_gravitational_parameter, frame_origin, frame_orientation )

File: docs/source/_src_snippets/simulation/environment_setup/environment_models/kepler_ephemeris.cpp
Eigen::Vector6d initialStateInKeplerianElements = ...

double epochOfInitialState = ...
double centralBodyGravitationalParameter = ...

std::string frameOrigin = "SSB";
std::string frameOrientation = "J2000";

bodySettings[ "Jupiter" ]->ephemerisSettings = std::make_shared< KeplerEphemerisSettings >(
    initialStateInKeplerianElements, epochOfInitialState, centralBodyGravitationalParameter, frameOrigin, frameOrientation );